src/vma/sock/sockinfo.h

```diff
@@ -97,6 +97,8 @@ struct cmsg_state
 	size_t		cmsg_bytes_consumed;
 };
 
+#define NOTIFY_ON_EVENTS(context, events) context->set_events(events)
```

Comment:
set_events will update some ring parameters even was socketxtreme is not used
we should change it ...

	if (!m_socketxtreme.ec.completion.events) {
					m_socketxtreme.ec.completion.user_data = (uint64_t)m_fd_context;
					m_p_rx_ring->put_ec(&m_socketxtreme.ec);
				}
				m_socketxtreme.ec.completion.events |= events;

Reply:
do you suggest check for `m_socketxtreme.active`

Reply:
done